tests/type-tests/test/openai.test-d.ts

```diff
@@ -18,9 +18,9 @@ expectType<Promise<OpenAiChatCompletionResponse>>(
 );
 
 /**
- * Chat Completion with invalid model.
+ * Chat Completion with custom model.
  */
-expectError(
+expectType(
```

Comment:
[q] Is this correct for OpenAI? If we have all models, it should not be possible to pass anything else, right? The string variant might only be needed for orchestration?

Reply:
During the implementation, I found out that for Azure openai, the model `gpt-4o-mini` was missing. That is why I thought, it could be the case that we could also miss some models when ai core added something there. Because no auto sync, I can imagine that sometime in the future, user will request us to support some new model from openai, and it could only be fixed via a new release. That's also why I used `LiteralUnion` for openai as well to support custom string.

Reply:
I think that is fair. If the model really is incorrect, you will receive a runtime error anyways. 👍